packages/shared/src/components/post/SquadPostAuthor.tsx

```diff
@@ -68,7 +68,7 @@ function SquadPostAuthor({
         <a
           href={author.permalink}
           className={classNames(
-            'ml-4 flex flex-1 flex-col overflow-hidden',
```

Comment:
There are only two places where we use this specific component. And on the other one, we pass the `flex-1` already, and the broken one does not need this.